# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/hnterm/hotfix-include.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- hnterm.git/src/hn-state.h.vanilla	2025-01-30 00:20:26.376666117 +0100
+++ hnterm.git/src/hn-state.h	2025-01-30 00:20:29.813332784 +0100
@@ -8,6 +8,7 @@
 #include <string>
 #include <vector>
 #include <variant>
+#include <cstdint>
 
 namespace HN {
 
